src/aggregators/directcr.jl

```diff
@@ -87,11 +84,11 @@ function initialize!(p::DirectCRJumpAggregation, integrator, u, params, t)
     # initialize rates
     fill_rates_and_sum!(p, u, params, t)
 
-    # if no maxrate was set, use largest initial rate (pad by 2 for an extra group)
-    isinf(p.maxrate) && (p.maxrate = 2*maximum(p.cur_rates))
```

Comment:
I went to merge, and noticed that it now appears we don't actually use the `p.maxrate` with the updates. Should we just remove that field from the aggregation? Otherwise this is good to go.

Reply:
Actually, we can remove this later on. Maybe we want to tweak this down the line to support using that field. So I'll merge as is now.